src/components/SQForm/SQFormMaskedReadOnlyField.tsx

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import MaskedInput from 'react-text-mask';
-import SQFormReadOnlyField from './SQFormReadOnlyField';
+import {maskProp} from 'types';
+import SQFormReadOnlyField, {
+  SQFormReadOnlyFieldProps,
+} from './SQFormReadOnlyField';
 
-function TextFieldMask({inputRef, mask, ...rest}) {
+interface TextFieldMaskProps extends React.HTMLAttributes<HTMLInputElement> {
+  inputRef?: (ref: HTMLElement | null) => void;
+  /** Valid mask array; custom or from utils/masks.js */
+  mask?: maskProp;
+}
+
+interface SQFormMaskedReadOnlyFieldProps extends SQFormReadOnlyFieldProps {
+  /** Valid mask array; custom or from utils/masks.js */
+  mask?: maskProp;
+  /** Name of the field will be the Object key of the key/value pair form payload */
+  name: string;
+  /** Descriptive label of the input */
+  label: string;
+  /** Placeholder text used inside the input field to provide hints to the user */
+  placeholder?: string;
+  /** Size of the input given full-width is 12. */
+  size?: 'auto' | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
```

Comment:
SQFormReadOnlyFieldProps extends BaseFieldProps which already defines `name`, `label`, and `size` for us so no need to do it here.

Reply:
@20BBrown14 Ahh I see, thanks for pointing that out.  Will update my PR.